encoder/src/main/scala/org/clulab/scala_transformers/encoder/apps/TokenClassifierExampleApp.scala

```diff
@@ -10,8 +10,8 @@ import org.clulab.scala_transformers.tokenizer.LongTokenization
 */
 
 object TokenClassifierExampleApp extends App {
-  //val tokenClassifier = TokenClassifier.fromFiles("../../scala-transformers-models/roberta-base-mtl/avg_export")
-  val tokenClassifier = TokenClassifier.fromFiles("../microsoft-deberta-v3-base-mtl/avg_export")
+  val tokenClassifier = TokenClassifier.fromFiles("/home/msurdeanu/github/scala-transformers/roberta-base-mtl/avg_export")
```

Comment:
Do you want to leave this directory in the source?

Reply:
Not necessarily :)